configs/eslint-plugin-compass/rules/no-leafygreen-outside-compass-components.js

```diff
@@ -78,12 +100,32 @@ module.exports = {
     },
     fixable: 'code',
     hasSuggestions: true,
-    // CWD for testing purposes only
-    schema: [{ type: 'string' }],
+
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          // Allows to override CWD for testing purposes
+          cwd: { type: 'string' },
+        },
+      },
+    ],
   },
   create(context) {
-    const [cwd] = context.options;
-    const packageName = getPackageNameFromCwd(cwd ?? context.getCwd());
+    const options = {
+      cwd: context.getCwd(),
+      ...context.options[0],
+    };
+
+    // Physical filenames like `<text>` or `<input>` indicate programmatic usage
+    // with no custom filename provided
+    //
+    // See: https://eslint.org/docs/developer-guide/working-with-rules#the-context-object
+    const dirname = /<.+?>/.test(context.getPhysicalFilename())
```

Comment:
not sure if it does make sense to do this here, but if it does:

```suggestion
    const dirname = /^<.+?>$/.test(context.getPhysicalFilename())
```

Reply:
Yeah, totally worth being a bit stricter here